CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Self-Assign button for Flaws
 * Provide time to Public Date field
 * Add neighboring dropdown menu to Flaw Description for its review workflow
+* Alerts on Flaws (OSIDB-1211)
```

Comment:
Are we adding task IDs on changelog?
I'd specify a bit more as we have the toasts and this can be misunderstood.
e.g.: Embedded alerts on Flaw forms

Reply:
@C-Valen fixed the description of the change. Regarding task IDs, I am not sure, I vaguely remember that we said we are going to mention them on changelog, but not sure what is the current consensus.

Reply:
Leaving Jira ID as per discussion in slack.